pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -225,3 +227,16 @@ def test_convert_bool_dtype(self):
         # GH32287
         df = pd.DataFrame({"A": pd.array([True])})
         tm.assert_frame_equal(df, df.convert_dtypes())
+
+    def test_convert_object_to_datetime_with_cache(self):
```

Comment:
can you put this with the other to_datetime tests: pandas/tests/tools/test_to_datetime.py; there a number of caching tests if you can locate nearby

Reply:
Moved, hope the location is ok, cache tests are everywhere there :)

Reply:
yep lgtm. lots of tests :->